### PR TITLE
fix(chips): allow undefined as passed ngModel and initialize the array

### PR DESCRIPTION
**Pull request by kevinmerckx**
Without this fix, giving an undefined variable in ngModel made the chips component fail as it tried to use the push method of an undefined variable. The array is now initialized if necessary.

### CODE REVIEW

File: src/components/chips/js/chipsController.js [hunk 1]
```diff
@@ -280,6 +280,11 @@ MdChipsCtrl.prototype.appendChip = function(newChip) {
   // Check for a null (but not undefined), or existing chip and cancel appending
   if (newChip == null || this.items.indexOf(newChip) + 1) return;
 
+	// Assign a value to the ngModel bound variable if it is not defined yet
+	if (!this.ngModelCtrl.$viewValue) {
+		this.ngModelCtrl.$setViewValue(this.items);
```
Comment (DevVersion) [inline src/components/chips/js/chipsController.js]: Please indent with `two` spaces, instead of a tab.

Comment (kevinmerckx) [inline src/components/chips/js/chipsController.js]: @DevVersion I'm not quite sure that the `$render` function is the best place for setting the `$viewValue`: the `ngModel` would be directly set to `[]`while I think we should wait for a chip to be appended to set the value to `[…]`

**Comment by kevinmerckx** [comment]
@DevVersion I've fixed the indentation.

**Comment by kevinmerckx** [comment]
@DevVersion any news for this PR?

**Comment by ThomasBurleson** [comment]
lgtm

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our ‘Surge Focus on Material 2' efforts.
        For details, see our [forum posting](http://bit.ly/1UhZyWs).

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our ‘Surge Focus on Material 2' efforts.
      For details, see our forum posting @ http://bit.ly/1UhZyWs.

**Pull request closed without merging**